docs/helm-deployment-minikube.md

```diff
@@ -0,0 +1,127 @@
+# Alfresco Content Services Community Deployment with Helm using Minikube
+
+## Prerequisites
+
+To run the Alfresco Content Services Community deployment using Minikube requires:
+
+| Component   | Getting Started Guide |
+| ------------| --------------------- |
+| Minikube    | https://github.com/kubernetes/minikube |
+| Kubectl     | https://kubernetes.io/docs/tasks/tools/install-kubectl/ |
+| Helm        | https://docs.helm.sh/using_helm/#quickstart-guide |
+
+### Starting Minikube VM
+
+* Allocate at least 16GB RAM (to distribute among the Alfresco Content Services Community cluster nodes), 2 CPU cores, and 20GB disk space to the Minikube VM:
```

Comment:
We don't support clustering in the Alfresco Community, so maybe the current line should be replace with:
"Allocate at least 12GB RAM,  4 CPU cores and 20GB disk space to the Minikube VM:"

I lowered the RAM to 12GB, as community should use less, since it does not have all the Transformer services and it can only have one node of repository